rocketpy/rocket/aero_surface/fins/free_form_fins.py

```diff
@@ -0,0 +1,379 @@
+        Fins cant angle with respect to the rocket centerline, in degrees.
+    FreeFormFins.changing_attribute_dict : dict
+        Dictionary that stores the name and the values of the attributes that
+        may be changed during a simulation. Useful for control systems.
+    FreeFormFins.cant_angle_rad : float
+        Fins cant angle with respect to the rocket centerline, in radians.
+    FreeFormFins.root_chord : float
+        Fin root chord in meters.
+    FreeFormFins.tip_chord : float
+        Fin tip chord in meters.
+    FreeFormFins.span : float
+        Fin span in meters.
+    FreeFormFins.name : string
+        Name of fin set.
+    FreeFormFins.sweep_length : float
+        Fins sweep length in meters. By sweep length, understand the axial
+        distance between the fin root leading edge and the fin tip leading edge
+        measured parallel to the rocket centerline.
+    FreeFormFins.sweep_angle : float
+        Fins sweep angle with respect to the rocket centerline. Must
+        be given in degrees.
+    FreeFormFins.d : float
+        Reference diameter of the rocket, in meters.
+    FreeFormFins.ref_area : float
+        Reference area of the rocket, in m².
+    FreeFormFins.Af : float
+        Area of the longitudinal section of each fin in the set.
+    FreeFormFins.AR : float
+        Aspect ratio of each fin in the set
+    FreeFormFins.gamma_c : float
+        Fin mid-chord sweep angle.
+    FreeFormFins.Yma : float
+        Span wise position of the mean aerodynamic chord.
+    FreeFormFins.roll_geometrical_constant : float
+        Geometrical constant used in roll calculations.
+    FreeFormFins.tau : float
+        Geometrical relation used to simplify lift and roll calculations.
+    FreeFormFins.lift_interference_factor : float
+        Factor of Fin-Body interference in the lift coefficient.
+    FreeFormFins.cp : tuple
+        Tuple with the x, y and z local coordinates of the fin set center of
+        pressure. Has units of length and is given in meters.
+    FreeFormFins.cpx : float
+        Fin set local center of pressure x coordinate. Has units of length and
+        is given in meters.
+    FreeFormFins.cpy : float
+        Fin set local center of pressure y coordinate. Has units of length and
+        is given in meters.
+    FreeFormFins.cpz : float
+        Fin set local center of pressure z coordinate. Has units of length and
+        is given in meters.
+    FreeFormFins.cl : Function
+        Function which defines the lift coefficient as a function of the angle
+        of attack and the Mach number. Takes as input the angle of attack in
+        radians and the Mach number. Returns the lift coefficient.
+    FreeFormFins.clalpha : float
+        Lift coefficient slope. Has units of 1/rad.
+    FreeFormFins.mac_length : float
+        Mean aerodynamic chord length of the fin set.
+    FreeFormFins.mac_lead : float
+        Mean aerodynamic chord leading edge x coordinate.
+    """
```

Comment:
It seems to me that some of these attributes do not make sense for free form fins. For instance, does `sweep_length` makes sense for `FreeFormFins`? It looked to me that it is something related to the geometry of a `TrapezoidalFins`.